File: specs/master/cluster/nodes_stats/node_jvm_stats.ts

/**namespace:Cluster.NodesStats */
interface node_jvm_stats {
	timestamp: long;
	uptime: string;
	uptime_in_millis: long;
	mem: memory_stats;
	threads: thread_stats;
	gc: garbage_collection_stats;
	/**custom_serialization */
	buffer_pools: map<string, node_buffer_pool>[];
	classes: jvm_classes_stats;
}

File: output/csharp/Cluster/NodesStats/NodeJvmStats.cs
using Nest.Cluster;
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cluster {

	public class NodeJvmStats  {
		
		[DataMember(Name="buffer_pools")]
		public IDictionary<string, NodeBufferPool> BufferPools { get; set; }


		[DataMember(Name="classes")]
		public JvmClassesStats Classes { get; set; }


		[DataMember(Name="gc")]
		public GarbageCollectionStats Gc { get; set; }


		[DataMember(Name="mem")]
		public MemoryStats Mem { get; set; }


		[DataMember(Name="threads")]
		public ThreadStats Threads { get; set; }


		[DataMember(Name="timestamp")]
		public long Timestamp { get; set; }


		[DataMember(Name="uptime")]
		public string Uptime { get; set; }


		[DataMember(Name="uptime_in_millis")]
		public long UptimeInMillis { get; set; }

	}
}
